back/engines/commercial/id_oostende_rrn/lib/id_oostende_rrn/wijk_budget_api.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+module IdOostendeRrn
+  class WijkBudgetApi
+    include HTTParty
+    debug_output $stdout if Rails.env.development? || Rails.env.test?
+
+    def initialize(api_key:, environment:)
+      @api_key = api_key
+      # Only one address provided by Oostende thus far.
+      @domain = case environment
+      when 'dv'
+        'wapaza-wijkprikkelsapi-01.azurewebsites.net'
+      when 'qa'
+        'wapaza-wijkprikkelsapi-01.azurewebsites.net'
+      when 'production'
+        'wapaza-wijkprikkelsapi-01.azurewebsites.net'
```

Comment:
`@domain` is set to the same value, whatever the case, as I have only received one API URL, as yet. I left the case statement in place on the assumption that we will later receive details of the other URLs, though I am not sure we will, or even if we need them, at this point.